incubator/README.md

```diff
@@ -0,0 +1,195 @@
+# PingCAP Incubator V1.0
+
+PingCAP Incubator is where all new features and projects associated with TiDB
+Ecosystem can get started. Once going through the incubation process, they can
+either be merged to tidb/tikv repo or become full PingCAP Community Projects.
+
+The process is designed to ensure that new features and projects have correct
+licensing, up-to-date boilerplate documents, a healthy community process, and
+are developed using accepted PingCAP Community practices.
+
+Welcome to join [TiDB Community Slack
+Workspace](https://pingcap.com/tidbslack/) and feel free to contact us in the
+**#incubator** channel if any question or suggestion arised.
+
+## Content
+
+* [What you are going to incubate?](#what-you-are-going-to-incubate)
+* [Features Incubation Process](#features-incubation-process)
+    * [Apply for Incubation](#apply-for-incubation)
+        * [Write a proposal](#write-a-proposal)
+        * [Find a Mentor](#find-a-mentor)
+        * [Get approval from PMC (Project Management Committee)](#get-approval-from-pmc-project-management-committee)
+        * [Complete team member recruitment](#complete-team-member-recruitment)
+    * [Start Incubation](#start-incubation)
+    * [Graduation](#graduation)
+* [Projects Incubation Process](#projects-incubation-process)
+    * [Apply for Incubation](#apply-for-incubation-1)
+        * [Write a proposal](#write-a-proposal-1)
+        * [Find a Mentor](#find-a-mentor-1)
+        * [Get approval of PMC (Project Management Committee)](#get-approval-of-pmc-project-management-committee-1)
+        * [Recruit team members](#Recruit-team-members)
+    * [Start Incubation](#start-incubation-1)
+    * [Maturity Level Promotion](#maturity-level-promotion)
+    * [Graduation](#graduation-1)
+    * [Retirement](#retirement)
+* [Benefits (WIP)](#benefits-wip)
+* [Features and Projects (WIP)](#features-and-projects-wip)
+
+<!-- vim-markdown-toc -->
+
+## What you are going to incubate?
+
+- If what you want to incubate is a feature that needs to be merged into an
+  existing project, please follow [Features Incubation Process](#features-incubation-process)
```

Comment:
```suggestion
  existing project on [`PingCAP` organization](https://github.com/pingcap), please follow [Features Incubation Process](#features-incubation-process)
```

Reply:
BTW, how about:
* s/Features Incubation Process/the process of Feature Incubator/
* s/Projects Incubation Process/the process of Project Incubator/

@dcalvin How do you think?